appinfo/info.xml

```diff
@@ -14,6 +14,15 @@
 
 	<default_enable/>
 
+	<documentation>
+		<user>https://doc.owncloud.org/missing</user>
+		<admin>https://doc.owncloud.org/missing</admin>
+		<developer>https://doc.owncloud.org/missing</developer>
```

Comment:
dev docs are in https://github.com/owncloud/notifications/tree/master/docs

others don't exist, so you may as well delete them